src/views/Home.jsx

```diff
@@ -1,11 +1,12 @@
 import { SingleList } from '../components/SingleList';
 import './Home.css';
-import { createList } from '../api/firebase';
+import { createList, deleteList } from '../api/firebase';
 import { useState } from 'react';
-import { useNavigate } from 'react-router-dom';
+import { Link, useNavigate } from 'react-router-dom';
 
 export function Home({ data, setListPath, userId, userEmail }) {
 	const [name, setName] = useState('');
+	const [selectedList, setSelectedList] = useState(null); // State to track the selected list
```

Comment:
You do not need this additional state here.  We already have the `listPath` in the `App`, you can pass it as props and use it.

Reply:
okay... thank you

Reply:
Fixed